.github/workflows/cd.yml

```diff
@@ -0,0 +1,56 @@
+name: Install deps, Test & Release
+on:
+  push:
+    branches: master
+jobs:
+  install-deps-test-release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository 
+      uses: actions/checkout@v2.4.0
+
+    - name: Set up Python
+      uses: actions/setup-python@v2.3.1
+      with:
+        python-version: 3.9.9
+        
+    - name: Set up Poetry
+      uses: abatilo/actions-poetry@v2.1.4
+      with:
+        poetry-version: 1.1.12
+
+    - name: Install dependencies
+      run: poetry install
+
+    - name: Test
```

Comment:
Why are we repeating testing in CD? Is it because Santoku is being published and it needs extra robustness to ensure everything works well?

Reply:
Good question. This last check ensures that mixing master/main branch with the PR doesn't break anything.